ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml

```diff
@@ -219,35 +219,93 @@
                     <h4>{{ warning.message }}</h4>
                 </li>
             </ul>
-            <validate-date ref="date0" name="Symptom Onset Date" model="ClinicalDetails" property="SymptomStartDate" rank="0" v-model="dates[0]" inline-template>
+
+            <nhs-form-group nhs-form-group-type="Standard">
+                <nhs-fieldset>
+                    <nhs-fieldset-legend nhs-legend-size="Standard">Patient is symptomatic</nhs-fieldset-legend>
+                    <div class="nhsuk-radios govuk-radios--conditional" data-module="govuk-radios">
+                        <div class="nhsuk-radios__item">
+                            <input asp-for="ClinicalDetails.IsSymptomatic" id="symptomatic-yes" class="nhsuk-radios__input" type="radio" value="true" data-aria-controls="conditional-symptomatic-conditional">
+                            <label class="nhsuk-label nhsuk-radios__label" for="symptomatic-yes">
+                                Yes
+                            </label>
+                        </div>
+                        <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-symptomatic-conditional">
+                                @{
+                                    var SymptomError = !Model.IsValid("ClinicalDetails.SymptomStartDate");
```

Comment:
Convention to camel(lower) c# variables, I've also been slowly shifting these towards being called `hasSymptomError`, don't know if I've got team support on this, but no one has been stopping me...